src/Mailgun/Model/MailingList/CreateMemberResponse.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/*
+ * Copyright (C) 2013 Mailgun
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+namespace Mailgun\Model\MailingList;
+
+use Mailgun\Model\ApiResponse;
+
+/**
+ * @author Michael Münch <helmchen@sounds-like.me>
+ */
+final class CreateMemberResponse implements ApiResponse
+{
+    /**
+     * @var Member
+     */
+    private $member;
+
+    /**
+     * @var string
+     */
+    private $message;
+
+    public static function create(array $data)
+    {
+        $member = null;
+
+        if (isset($data['member'])) {
+            $member = Member::create($data['member']);
+        }
```

Comment:
Although this is not a blocker, to be consistent with the rest of the API, I would suggest to define the `$member` variable in a single row:

```php
$member = isset($data['member']) ? Member::create($data['member']) : null;
```

Same case for any other parameter handled by Model and Response files, so we will keep the same logic and format everywhere.